config/locales/activemodel.en.yml

```diff
@@ -13,3 +13,4 @@ en:
             csv_file:
               invalid_extension: "is invalid, must have .csv extension."
               invalid_count: "is invalid, must have 1-1000 keywords."
+      not_found: "No %{model_name} found."
```

Comment:
Since this is now a general ActiveRecord error, it should be in the locale file `config/locales/activerecord.en.yml` (see again: https://github.com/nimblehq/ewa-payroll-web/blob/develop/config/locales/activerecord.en.yml).

To recap:
- Use `activerecord.en.yml` for all things related to `app/models` (so database-backed)
- Use `activemodel.en.yml` for all things related to POROs (e.g., form objects)